specification/Metadata/Semantics/README.md

```diff
@@ -76,115 +37,44 @@ Units for all linear distances are meters, and all angles are radians.
-
-The horizon occlusion point of the tile expressed in an ellipsoid-scaled fixed frame. If this point is below the horizon, the entire tile is below the horizon. See [Horizon Culling](https://cesium.com/blog/2013/04/25/horizon-culling/) for more information.
-
-* Type: `VEC3`
-* Component type: `FLOAT32` or `FLOAT64`
+Semantic|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Type&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|Description
+--|--|--
+`TILE_BOUNDING_BOX`|<ul><li>Type: `ARRAY`</li><li>Component type: `FLOAT32` or `FLOAT64`</li><li>Component count: `12`</li></ul>|The bounding volume of the tile, expressed as a [box (as defined by 3D Tiles 1.0)](../../../specification#box). Equivalent to `tile.boundingVolume.box`.
+`TILE_BOUNDING_REGION`|<ul><li>Type: `ARRAY`</li><li>Component type: `FLOAT64`</li><li>Component count: `6`</li></ul>|The bounding volume of the tile, expressed as a [region (as defined by 3D Tiles 1.0)](../../../specification#region). Equivalent to `tile.boundingVolume.region`.
+`TILE_BOUNDING_SPHERE`|<ul><li>Type: `ARRAY`</li><li>Component type: `FLOAT32` or `FLOAT64`</li><li>Component count: `4`</li></ul>|The bounding volume of the tile, expressed as a [sphere (as defined by 3D Tiles 1.0)](../../../specification#sphere). Equivalent to `tile.boundingVolume.sphere`.
+`TILE_MINIMUM_HEIGHT`|<ul><li>Type: `SINGLE`</li><li>Component type: `FLOAT32` or `FLOAT64`</li></ul>|The minimum height of the tile above (or below) the WGS84 ellipsoid. Equivalent to minimum height component of `TILE_BOUNDING_REGION` and `tile.boundingVolume.region`.
+`TILE_MAXIMUM_HEIGHT`|<ul><li>Type: `SINGLE`</li><li>Component type: `FLOAT32` or `FLOAT64`</li></ul>|The maximum height of the tile above (or below) the WGS84 ellipsoid. Equivalent to maximum height component of `TILE_BOUNDING_REGION` and `tile.boundingVolume.region`.
+`TILE_HORIZON_OCCLUSION_POINT`|<ul><li>Type: `VEC3`</li><li>Component type: `FLOAT32` or `FLOAT64`</li></ul>|The horizon occlusion point of the tile expressed in an ellipsoid-scaled fixed frame. If this point is below the horizon, the entire tile is below the horizon. See [Horizon Culling](https://cesium.com/blog/2013/04/25/horizon-culling/) for more information.
+`TILE_GEOMETRIC_ERROR`|<ul><li>Type: `SINGLE`</li><li>Component type: `FLOAT32` or `FLOAT64`</li></ul>|The geometric error of the tile. Equivalent to `tile.geometricError`.
 
 > **Implementation Note**: `TILE_HORIZON_OCCLUSION_POINT` should account for all content in a tile and its descendants, whereas `CONTENT_HORIZON_OCCLUSION_POINT` should only account for content in a tile. When the two values are equivalent, only `TILE_HORIZON_OCCLUSION_POINT` should be specified.
```

Comment:
```suggestion
<small><sup>1</sup> `TILE_HORIZON_OCCLUSION_POINT` should account for all content in a tile and its descendants, whereas `CONTENT_HORIZON_OCCLUSION_POINT` should only account for content in a tile. When the two values are equivalent, only `TILE_HORIZON_OCCLUSION_POINT` should be specified.</small>
```

Minor nit, but on re-reading this probably shouldn't be an implementation note – it's part of the meaning of the semantic. Calling it out as an end-note attached to `TILE_HORIZON_OCCLUSION_POINT`  may be clearer.